Refactor footer to allow multicolumn and external links (#904)
openfun/richie: UPGRADE.md

## Unreleased

## 1.13.x to 1.14.x

> - Footer links are now defined via a static placeholder (see CHANGELOG). We provide a data

Reviewer: ```suggestion
- Footer links are now defined via a static placeholder (see CHANGELOG). Richie provides a data
```

>   migration to make sure that your existing footer of flat links is ported to the new static

Reviewer: ```suggestion
  migration to make sure that an existing footer of flat links is ported to the new static
```

>   placeholder. Before running the data migration, make sure all your changes on the footer pages

Reviewer: ```suggestion
  placeholder. Before running the data migration, make sure all changes on the footer pages
```

  (pages that are below the page with a reverse_id of "annex") are published. After running the

>   data migration, check the footer on your site (draft and public) to make sure it is as expected.

Reviewer: ```suggestion
  data migration, check the footer (draft and public) to make sure it is as expected.
```
